### ISSUE TITLE
Add hotkey to clear all counters/notifications

### ISSUE THREAD

**Issue by nicoulaj** [thelounge/lounge#811]
It would be nice to have a keyboard shortcut to clear all notifications/counters/favicon.
<kbd>CTRL</kbd>+<kbd>L</kbd> is a commonly used "clear" shortcut.

**Comment by dgw** [comment]
<kbd>CTRL</kbd>+<kbd>L</kbd> is used in my client for moving the marker [b]l[/b]ine, so I would find that particular choice confusing.

But clearing all unread badges would be a nice thing to have, since I use The Lounge as a secondary client and often don't check it for a few days.

**Comment by nicoulaj** [comment]
<kbd>CTRL</kbd>+<kbd>SHIFT</kbd>+<kbd>L</kbd> maybe then ?

**Comment by astorije** [comment]
I don't think a shortcut for that would be best. I think it would be more appropriate as a UI element in #791. Then we can think configurable hotkeys as a whole as this is very specific IMO.

**Comment by nicoulaj** [comment]
I don't think this is very specific, usually software that comes with notifications also comes with an easily accessible way to clear it.
But I would be fine with a configurable hotkey.

**Comment by z0rc** [comment]
Slack and Discord both provide <kbd>Shift</kbd>+<kbd>Esc</kbd> as shortcut to clear all notification counters. Given the variety of opinions I believe making this shortcut configurable is the best approach.

**Comment by richrd** [comment]
I would suggest having a context menu item on channels and networks, something like "Mark all messages as read"
